scripts/build-sidecar.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+if [ "${SKIP_SIDECAR_BUILD}" == "true" ]; then
+  echo "Skipping Sidecar Build because SKIP_SIDECAR_BUILD was set to 'true'"
+else
+  SCRIPT_DIR=$( dirname -- "$( readlink -f -- "$0"; )"; )
+  SIDECAR_REPO="${SIDECAR_REPO:-https://github.com/apache/cassandra-sidecar.git}"
+  SIDECAR_BRANCH="${SIDECAR_BRANCH:-trunk}"
+  SIDECAR_JAR_DIR="$(dirname "${SCRIPT_DIR}/")/dependencies"
+  SIDECAR_BUILD_DIR="${SIDECAR_JAR_DIR}/sidecar-build"
+  JAVA_MAJOR_VERISON="$(javap -verbose java.lang.String | grep "major version" | cut -d " " -f5)"
+  echo $JAVA_MAJOR_VERISON
+  if [ "${JAVA_MAJOR_VERISON}" == "52" ]; then
+    SIDECAR_BUILD_VERSION="1.0.0-jdk8-analytics"
```

Comment:
nit: how about naming the versions as `1.0.0-jdk8-analytics-dev` and `1.0.0-analytics-dev` respectively?

Reply:
The reason _not_ to do this is so that local developers can pick these up without having to change all of the profiles - with the way we set up the `mavenLocal` directory you can just pick up whatever is local without having to change anything - otherwise, we have to add a bunch of mechanics to pick local dev vs. pulled from Maven... if you have nothing in the `dependencies` folder, it uses what's in maven - otherwise it'll find it locally without modification. And since there _is no maven_ publishing, there's no other source for this anyway.